zelda3.ini

```
@@ -5,6 +5,9 @@
 # This default is suitable for QWERTY keyboards.
 Controls = Up, Down, Left, Right, Right Shift, Return, x, z, s, a, d, c
 
+# This default is suitable for QWERTZ keyboards.
+Controls = Up, Down, Left, Right, Right Shift, Return, x, y, s, a, d, c

```

Shouldn't that be commented by default?
```suggestion
#Controls = Up, Down, Left, Right, Right Shift, Return, x, y, s, a, d, c
```